<launch>
    <arg name="pose_topic" default=""/>

    <node name="landing_controller" pkg="quadrotors_landing_vision" type="landing_controller.py" respawn="false" output="screen">
        <rosparam file="$(find quadrotors_landing_vision)/config/drone/landing_controller.yaml" command="load"/>
        <param name="pose_topic" value="$(arg pose_topic)"/>
    </node>

</launch>
